CONTRIBUTING.md

```diff
@@ -74,9 +74,9 @@ These dependencies are made available in different ways depending on context.
 ### Local Development
 
 When developing locally, you can download all of these dependencies by running
-`npm run init`. This provides the following options for `compiler` (for the
-embedded compiler), `protocol` (for the embedded protocol), and `api` (for the
-JS API):
+`npm install` and then `npm run init`. This provides the following options for
+`compiler` (for the embedded compiler), `protocol` (for the embedded protocol),
+and `api` (for the JS API):
```

Comment:
The content change here is to add "`npm install` and then ". The rest of the diff is reflowing to 80 characters.